k8s/crds/kops.k8s.io_clusters.yaml

```
@@ -1508,6 +1508,10 @@ spec:
                 properties:
                   enabled:
                     type: boolean
+                  logFormat:
```

To keep naming consistent with official manifest.
```suggestion
                  logEncoding:
```

---

Done :+1: